src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import core.basesyntax.service.FileReader;
+import core.basesyntax.service.FileWriter;
+import core.basesyntax.service.ReportGenerator;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.impl.DataConverterImpl;
+import core.basesyntax.service.impl.FileReaderImpl;
+import core.basesyntax.service.impl.FileWriterImpl;
+import core.basesyntax.service.impl.ReportGeneratorImpl;
+import core.basesyntax.service.impl.ShopServiceImpl;
+import core.basesyntax.strategy.BalanceOperation;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.PurchaseOperation;
+import core.basesyntax.strategy.ReturnOperation;
+import core.basesyntax.strategy.SupplyOperation;
+import core.basesyntax.strategy.impl.OperationStrategyImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] arg) {
+        final String inputFilePath = "src/main/resources/reportToRead.csv";
+        final String outputFilePath = "src/main/resources/finalReport.csv";
```

Comment:
these fields should be constants (class-level)
```suggestion
        final String inputFilePath = "src/main/resources/reportToRead.csv";
        final String outputFilePath = "src/main/resources/finalReport.csv";
```

Reply:
fixed 2e6a01f

Reply:
not fixed